File: atelier_couture/src/data/entites/Fournisseur.java
package data.entites;

public class Fournisseur extends Personne{

    private String telFixe;

    public Fournisseur() {}
    public Fournisseur(String nom,String prenom,String telephone,String adresse){
        super(nom,prenom,telephone,adresse);
    }

    public String getTelFixe() {
        return telFixe;
    }

    public void setTelFixe(String telFixe) {
        this.telFixe = telFixe;
    }

    @Override
    public String toString() {
        return "Fournisseur{" +
                "id=" + id +
                ", telFixe='" + telFixe + '\'' +
                '}';
    }
}

File: model/Fournisseur.php
<?php

require_once "Personne.php";

class Fournisseur extends Personne{

    private String $telFixe;
    

    public function getTelFixe()
    {
        return $this->telFixe;
    }


    public function setTelFixe($telFixe)
    {
        $this->telFixe = $telFixe;

        return $this;
    }

    public static function toFournisseur($row):Fournisseur{
        $fournisseur=new Fournisseur();
        $fournisseur->setId($row['id']);
        $fournisseur->setNom($row['nom']);
        $fournisseur->setPrenom($row['prenom']);
        $fournisseur->setAdresse($row['adresse']);
        $fournisseur->setTelephone($row['tel']);
        $fournisseur->setTelFixe($row['telFixe']);
        return $fournisseur;

    }

}